webpack.config.js

```diff
@@ -158,21 +171,8 @@ function getWebpackConfig( {
 			runtimeChunk: codeSplit ? { name: 'manifest' } : false,
 			moduleIds: 'named',
 			chunkIds: isDevelopment ? 'named' : 'natural',
-			minimize: shouldMinify,
-			minimizer: [
-				new TerserPlugin( {
-					cache: process.env.CIRCLECI
-						? `${ process.env.HOME }/terser-cache`
-						: 'docker' !== process.env.CONTAINER,
-					parallel: workerCount,
-					sourceMap: Boolean( process.env.SOURCEMAP ),
-					terserOptions: {
-						ecma: 5,
-						safari10: true,
-						mangle: calypsoEnv !== 'desktop',
-					},
-				} ),
-			],
+			minimize: minimizerConfig.minimize,
```

Comment:
Let's just use `minimize: shouldMinify` directly here so we don't have to take the detour through `minimizerConfig`.

Reply:
good call!